lib/utils/utils.ts

```diff
@@ -5,6 +5,8 @@ import { createHash, randomBytes as cryptoRandomBytes } from 'crypto';
 import { promisify } from 'util';
 import moment from 'moment';
 
+const createKeccakHash = require('keccak');
```

Comment:
Rather than use `require` here, I think it'd be better to try and stay consistent and use `import` instead. I know that it'll give us an error because no type declarations are available for the module, but you can use `@ts-ignore` like so:

```typescript
// @ts-ignore
import createKeccakHash from 'keccak';
```